### PR TITLE
Google Chart QR code (Independent Publisher)

### PR DESCRIPTION
**Pull request by Ionavoss**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

flowruns
Test operation

### CODE REVIEW

**Comment by Ionavoss** [comment]
@microsoft-github-policy-service agree [company="Avanade"]

**Comment by Ionavoss** [comment]
@microsoft-github-policy-service agree

**Comment by Amjed-Ayoub** [comment]
Hello @Ionavoss,
Thank you so much for submitting your connector with us. if you would like us to review your pull request, please remove the "Proposal" from the title.
Please let me know if you have any questions.
Thank you so much.

**Comment by Ionavoss** [comment]
> Hello @Ionavoss, Thank you so much for submitting your connector with us. if you would like us to review your pull request, please remove the "Proposal" from the title. Please let me know if you have any questions. Thank you so much.

Thanks @Amjed-Ayoub, I just updated the title to remove the Proposal section :)

File: independent-publisher-connectors/Google Chart QR Code/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Google Chart QR Code Generator",
+    "description": "This Connector is build arount the Google Chart API to generate QR codes to be used in Power Platform applications",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Google Chart QR Code/apiDefinition.swagger.json:5]: @Ionavoss please remove the word "API" from the description. please follow our requirements [here ](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info).

@Ionavoss All files should be free of grammatical and spelling errors. in the description you have "This Connector is build arount the Google Chart...", maybe you meant to say "This Connector is **built around** the Google Chart..."? Can you please fix?

File: independent-publisher-connectors/Google Chart QR Code/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    "version": "1.0",
+    "contact": {
+      "name": "Iona Varga",
+      "email": "ionavarga@outlook.com"
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Google Chart QR Code/apiDefinition.swagger.json:9]: @Ionavoss is it possible to add a "url" property for your website?

File: independent-publisher-connectors/Google Chart QR Code/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "x-ms-connector-metadata": [
+  {
+      "propertyName": "Website",
+      "propertyValue": "https://chart.googleapis.com/"
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Google Chart QR Code/apiDefinition.swagger.json:22]: @Ionavoss please provide a valid URL. This is throwing 404 error. 
image

File: independent-publisher-connectors/Google Chart QR Code/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+                  "type": "string",
+                  "description": "$content"
+                }
+              }
+            }
+          }
+        },
+        "operationId": "GenerateQR",
+        "summary": "Generate a QR code based on a set of inputs",
+        "description": "By providing a width and heigth, data, encoding and correction level, the Google Chart API can generate a QR code.",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Google Chart QR Code/apiDefinition.swagger.json:56]: @Ionavoss All files should be free of grammatical and spelling errors. in the description you have "By providing a width and heigth, data...", maybe you meant to say "By providing a width and **height**, data..."? Can you please fix?

File: independent-publisher-connectors/Google Chart QR Code/readme.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+# Publisher: Iona Varga
+[My Profile](https://www.linkedin.com/in/iona-v-641487191/)
+
+# Prerequisites
+This connector doesn't require authentication. It's made available for free by Google.
+
+# Operations
+This connector only has the Generate QR code action
+
+Generate QR Code: Create a QR code based around a few parameters and some data input.
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Google Chart QR Code/readme.md:16]: @Ionavoss All files should be free of grammatical and spelling errors. in the description you have "Create a QR code based around a few...", maybe you meant to say "Create a QR code based **on** a few..."? Can you please fix?

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @Ionavoss,
I hope you are doing well.
I am following up with you to see how things are going with preparing your connector.
Please let us know if you have any questions.
Thank you.

**Comment by Amjed-Ayoub** [comment]
Hello @Ionavoss,
I hope you are doing well.
I am following up on my messages to see if you need any help with updating your connector artifacts.
Please do not hesitate to reach out to me if you have any questions as we are always here to help. 😊
I look forward to hearing from you.
Thank you very much.

**Comment by Amjed-Ayoub** [comment]
Hello @Ionavoss,
This is to follow up on the status of your connector. I would like to offer help to you if you are having any issues with your connector.
Please do not hesitate to reach out if you have any questions as we are always happy to help.
Thank you very much for working with us.

**Comment by Amjed-Ayoub** [comment]
Hello @Ionavoss ,

I hope you are doing well.

We would like to know the status of your connector. It has been in our repo since December 2022.
Please let us know how you would like to proceed.

Please do not hesitate to reach out if you have any questions as we are always happy to help.

Thank you very much.

**Comment by vmanoharas** [comment]
Hello @Ionavoss, 

I hope you are doing well.

We would like to know the status of your connector. It has been in our repo since December 2022.
Please let us know how you would like to proceed. 

Kindly note, this would be a last gentle reminder before we close this PR.

**Comment by vmanoharas** [comment]
Hello @lonavoss, 

This PR is open from more than 2 years, kindly create a new PR, I will close this PR.

**Pull request closed without merging**